src/main/java/com/manufacturing/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,203 @@
+        } catch (SQLException e) {
+            throw new DataProcessingException("Delete of car with id="
+                    + carId + " is failed", e);
+        }
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String query = "SELECT car_id, c.manufacturer_id, model, manufacturer_name, origin "
+                + "FROM cars c\n"
+                + "INNER JOIN manufacturers m ON c.manufacturer_id = m.manufacturer_id\n"
+                + "WHERE c.deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            ResultSet resultSet = statement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                cars.add(getCar(resultSet, connection));
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Something going wrong"
+                    + " with getting all cars", e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT cd.car_id, model, c.manufacturer_id, manufacturer_name, origin\n"
+                + "FROM cars_drivers cd\n"
+                + "INNER JOIN cars c ON cd.car_id = c.car_id\n"
+                + "INNER JOIN manufacturers m ON c.manufacturer_id = m.manufacturer_id\n"
+                + "WHERE cd.driver_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                cars.add(getCar(resultSet, connection));
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Something going wrong"
+                    + " with getting all drivers from car:" + driverId, e);
+        }
+    }
+
+    private void deleteDriversByID(Long carId) {
+        String query = "DELETE FROM cars_drivers WHERE car_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, carId);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("We can't delete drivers from list", e);
+        }
+    }
+
+    private List<Driver> getDrivers(Long carId, Connection connection) {
+        String query = "SELECT cd.driver_id, name, license_number\n"
+                + "FROM cars_drivers cd\n"
+                + "INNER JOIN drivers d\n"
+                + "\tON d.driver_id = cd.driver_id\n"
+                + "WHERE cd.car_id = ? AND deleted = false";
```

Comment:
deleted of which table?

Reply:
done, but as for me it is redundant because we have one table with `deleted` field

Reply:
It's better to specify the table because for someone who does't know the structure of your DB it is hard to understand what `deleted` relates to